tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/bottom_bar.py

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import PageRegion
+
+
+class BottomBar(PageRegion):
+    _bottom_bar_locator = (By.ID, 'bottombar')
+    _remove_locator = (By.ID, 'remove')
+    _done_locator = (By.ID, 'done')
+
+    def __init__(self, marionette):
+        PageRegion.__init__(self, marionette, self._bottom_bar_locator)
```

Comment:
PageRegion doesn't accept locators currently, that's https://bugzilla.mozilla.org/show_bug.cgi?id=1203527

I think you should do something similar like here:
http://mxr.mozilla.org/gaia/source/tests/python/gaia-ui-tests/gaiatest/apps/system/regions/fullscreen_dialog.py#20

So: 
```
root = Wait(marionette).until(
    expected.element_present(*self._bottom_bar_locator))
Wait(marionette).until(expected.element_displayed(root))
PageRegion.__init__(self, marionette, root)
```
That way, when accessing BottomBar we're sure that it is ready to be used.